### DISCUSSION TITLE
Series-communication-markdown Error

### DISCUSSION THREAD

**Discussion by lukepadiachy** [community/community#118858] [GitHub Education]
### Hi everyone,

# Issue 
<p>Completed the steps for this. but gives an error stating that it is unable to find my repo in the classroom , yet my repo says congratulations? Any help with this ? Thanks a mil :) </p>

![Screenshot_20240413_000635_Chrome](https://github.com/community/community/assets/148848073/7eb781ce-6463-42a4-b658-41bac73b679f)

![Screenshot_20240413_000645_Chrome](https://github.com/community/community/assets/148848073/9b32f4bd-1514-434d-a0aa-b4420ca63fe7)

**Comment by Flavio13Lins** [comment]
Hi @lukepadiachy .
Seems that it's a common issue, but it does not stop your learning process in GitHub Classromm.
Access https://education.github.com/pack/offers to see more tools of your interest and keep learning! 😊
